app/views/govuk_publishing_components/component_guide/show.html.erb

```diff
@@ -1,30 +1,48 @@
 <% content_for :title, "#{@component_doc.name} component" %>
 <%= render 'govuk_component/title', title: @component_doc.name, context: 'Component' %>
 
-<p class="component-description">
-  <%= @component_doc.description %>
-</p>
-
 <div class="component-show">
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <p class="component-description">
+        <%= @component_doc.description %>
+      </p>
+      <% if @component_doc.body.present? %>
```

Comment:
We're referencing @component_doc.html_body below, should this conditional be the same?